### DISCUSSION TITLE
Any way to run Mastodon with MySQL?

### DISCUSSION THREAD

**Discussion by michaelzap** [mastodon/mastodon#21124] [Q&A] [answered]
Howdy -

I'm already using a cloud hosting provider that I really like, and I have a Digital Ocean, Mailgun, and other services already set up there. I'd like to set up a Mastodon instance there as well, but this provider only offers MySQL database integration. Any way to work around this?

Thanks in advance for your help!

**Comment by ghost** [comment, marked as answer]
I don't think that you can, but I know that you can run MySQL and PostgreSQL at the same time.
